app/resources/views/home.html.twig

```diff
@@ -0,0 +1,25 @@
+{% extends "base.html.twig" %}
+
+{% block content_top %}
+    {% include "partials/hero.html.twig" with {
+        image_url: '/images/hero/home.jpg',
+    } %}
+{% endblock %}
+
+{% block content %}
+    <h1 class="title">PHP meetups in the south west, UK</h1>
+
+    <div class="md:flex -mx-4">
+        <div class="px-4 content leading-normal text-sm mb-6 md:w-2/3 md:mb-0">
+            <p>PHP User Group, meets 2nd Wednesday of every month, in Bristol, UK, talking PHP and related technologies. We typically have a 2 or 3 speakers each month, and cover topics like debugging, frameworks, packages, scaling & testing. Anyone of any skill level is welcome to attend and speak, we usually have beer, pizza and a projector available.</p>
+            <p>Find out more at <a href="http://www.meetup.com/php-sw">meetup.com/php-sw</a> and note our <a href="#0">code of conduct</a>.</p>
```

Comment:
Tiny thing. Links should be to https version of meetup. I'd probably grep for `http://` and update to `https://`

Reply:
I've updated these in-place. Ideally I'd have used `{{ info.meetupUrl }}` but it looks like it does need to be made global so it's available in all of the templates like the navigation and footer.